src/parseTools.js

```diff
@@ -1495,6 +1495,13 @@ function makeRetainedCompilerSettings() {
   return ret;
 }
 
+function guardAgainstUnincludedFunction(name) {
+  if (!addedLibraryItems[name]) {
+    return 'function ' + name + '() { abort("ccall is now in the JS library. to use it, build with -s EXTRA_LIBRARY_FUNCS_TO_INCLUDE=[' + "'$ccall'" + ']") }\n';
```

Comment:
Should this be `abort(name + " is now in the JS library ...`? This checks for `name`, but then issues a warning about `ccall`?

The name of this function feels ambiguous, does `guardAgainstUnincludedFunction` meant `guardAgainstUnincludedJsLibraryFunction`? Or `guardAgainstUnincludedPreambleFunction?`